### PR TITLE
Fix/208 add a active class to links in header

### PR DESCRIPTION
**Pull request by gazaston**
Adds the [`active_link_to` gem](https://github.com/comfy/active_link_to) for active nav highlighting.

screen shot 2018-06-25 at 15 37 06

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -24,7 +24,7 @@
           = t('app.title')
         %ul#proposition-links
           - if authenticated?
-            %li= link_to t('nav.school_page_link'), new_sessions_path
+            %li= active_link_to t('nav.school_page_link'), new_sessions_path, active: [['hiring_staff/schools'], []]
```
Comment (despo) [inline app/views/layouts/application.html.haml]: This is such a small check that adding another depency seems unecessary.

We could just add a helper that returns 'active' or ''

```
def active_link_class(link_path)
    current_page?(link_path) ? 'active' : ''
end
```
### Usage
```
link_to t('nav.school_page_link'), new_sessions_path, class: active_link_class(school_path)
```

**Comment by gazaston** [comment]
[Add a .active class to links in header](https://trello.com/c/rrgH558X/208-add-a-active-class-to-links-in-header)

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (gazaston) [reply app/views/layouts/application.html.haml]: I had no idea it could be that simple 😊

**Comment by despo** [comment]
Do not merge!

**Pull request merged**